File: engine/model/src/main/scala/tools/packet/monster/carnival/MonsterCarnivalMessage.scala
package tools.packet.monster.carnival

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MonsterCarnivalMessage(private var _message: Byte) extends PacketInput {
  def message: Byte = _message

  override def opcode(): SendOpcode = SendOpcode.MONSTER_CARNIVAL_MESSAGE
}

File: engine/engine-model/src/main/java/tools/packet/monster/carnival/MonsterCarnivalMessage.java
package tools.packet.monster.carnival;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MonsterCarnivalMessage(Byte message) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MONSTER_CARNIVAL_MESSAGE;
   }
}